classes/LocalizationPack.php

```diff
@@ -402,13 +402,14 @@ protected function _installLanguages($xml, $install_mode = false)
             foreach ($xml->languages->language as $data) {
                 /** @var SimpleXMLElement $data */
                 $attributes = $data->attributes();
+                $isoCode = (string) $attributes['iso_code'];
                 // if we are not in an installation context or if the pack is not available in the local directory
```

Comment:
```suggestion
                // if we are not in an installation context or if the pack is available in the local directory
```

The comment seems wrong compared to what checks are performed no?

Reply:
Approved.